mixer/adapter/model/v1beta/value_type.proto

```
@@ -0,0 +1,59 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package istio.mixer.adapter.model.v1beta;
```

One of the protos in this package needs to have a package-level comment. This will be used at the top of the page on istio.io.

Additionally, one of the protos needs to have a comment that contains $location, $title, and $overview which are used to drive code gen. See other protos in istio/api for examples.

---

Will fix it. Do you think we should fail the BUILD if the package is missing these necessary stuff. I remember you already  have warnings for that stuff in your tool.

---

D